homeassistant/components/mqtt/strings.json

```
@@ -51,13 +51,35 @@
           "transport": "MQTT transport",
           "ws_headers": "WebSocket headers in JSON format",
           "ws_path": "WebSocket path"
+        },
+        "data_description": {
+          "broker": "The hostname or IP address of your MQTT broker.",
+          "port": "The port your MQTT broker listens to. For example 1883.",
+          "username": "The username to login to your MQTT broker.",
+          "password": "The password to login to your MQTT broker.",
+          "advanced_options": "Enable and click `next` to set advanced options.",
+          "certificate": "The custom CA certificate file to validate your MQTT brokers certificate.",
+          "client_id": "The unique ID to identify the Home Assistant MQTT API as MQTT client. It is recommended to leave this option blank.",
+          "client_cert": "The client certificate to authenticate against your MQTT broker.",
+          "client_key": "The private key file that belongs to your client certificate.",
+          "keepalive": "The time between sending keep alive messages",
```

```suggestion
          "keepalive": "The time between sending keep alive messages in seconds.",
```
Does this make sense?